fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordMetaData.java

```diff
@@ -73,6 +73,8 @@
     private final boolean splitLongRecords;
     private final boolean storeRecordVersions;
     private final int version;
+    private final long indexSubspaceKeyCounter;
+    private final boolean indexCounterBasedSubspaceKey;
```

Comment:
Should the names here match the names of their counterparts in the meta-data proto definition?